src/main/java/com/cleancommunity/user/MysqlUserDAO.java

```diff
@@ -71,10 +72,22 @@ public Boolean deleteUser(User user) {
 		// Returns true if query is valid and username exists
 		return this.updateQuery(sql_query);
 	}
+	public Boolean updateUser(User user) {
+		if (user instanceof Volunteer) {
+			String sql_query = String.format(
+					"UPDATE %s SET %s = %s, %s = %s, %s = %s WHERE username = \"%s\" AND password = \"%s\"",
+					TABLE_NAME, "completed", ((Volunteer)user).getProjectsCompleted(),
+					"inProgress", ((Volunteer)user).getProjectsCompleted(), "posted",
+					((Volunteer)user).getProjectsPosted(), user.getUsername(), user.getPassword());
```

Comment:
Just curious, but what is the importance of casting if user is already checked to be an instanceof volunteer?